go/vt/topo/server.go

```diff
@@ -176,6 +174,19 @@ var (
 	}
 )
 
+func init() {
+	for _, cmd := range []string{"vttablet", "vtctl", "vtctld", "vtcombo", "vtexplain", "vtgate",
+		"vtgr", "vtorc", "vtbackup"} {
```

Comment:
are the binaries for each file all the same? if so we should put them in one variable so we don't inevitably forget to update just one instance if we ever change this

Reply:
are you suggesting to have a constant [] across lock and server file?